packages/lesswrong/lib/collectionUtils.js

```diff
@@ -114,4 +114,18 @@ export function schemaDefaultValue(defaultValue) {
     onUpdate: throwIfSetToNull,
     canAutofillDefault: true,
   }
+}
+
+export function addUniversalFields({ collection }) {
+  collection.addField([
+    {
+      fieldName: 'schemaVersion',
+      fieldSchema: {
+        type: Number,
+        canRead: ['guests'],
+        ...schemaDefaultValue(1)
```

Comment:
It looks like `schemaDefaultValue` doesn't set `optional:true`, but it should (or else this should set it instead), so unit tests fail on form submissions (since it gets filled in server side).

Reply:
Thinking about this, I think it does potentially sometimes make sense to have things with `schemaDefaultValue` which are nevertheless non-optional from a form-submission perspective, so the `optional:true` should be on the field, rather than in the `schemaDefaultValue` function.

```suggestion
        optional: true,
        ...schemaDefaultValue(1)
```